ceph.spec.in

```diff
@@ -466,7 +488,17 @@ Requires:	librbd1 = %{epoch}:%{version}-%{release}
 Requires:	python-rados = %{epoch}:%{version}-%{release}
 Obsoletes:	python-ceph < %{epoch}:%{version}-%{release}
 %description -n python-rbd
-This package contains Python libraries for interacting with Cephs RADOS
+This package contains Python 2 libraries for interacting with Cephs RADOS
```

Comment:
Yes. There are contexts where the summary is the only thing the user sees. It seems to me that adding the " 2" or " 3" to the summary only increases the length by two characters, yet adds significant information value. It's just an idea - I don't feel strongly about it.